dali/base/dasds.cpp

```diff
@@ -3744,7 +3754,14 @@ int CSDSTransactionServer::run()
             mb.clear();
             if (coven.recv(mb, RANK_ALL, MPTAG_DALI_SDS_REQUEST, NULL))
             {
-                msgCount++;
```

Comment:
Removed by mistake...

Reply:
not intentional, but never used, but I'll put back, to be removed by a future PR

Reply:
reinstated